app/models/miq_request.rb

```diff
@@ -191,12 +191,15 @@ def must_have_user
   end
 
   def call_automate_event_queue(event_name)
+    user = User.current_user
```

Comment:
@lfu This might not be guaranteed every time you have to raise an error if the User.current_user is not set